src/common/logo.c

```diff
@@ -249,6 +269,37 @@ static void loadGarudaLogo(FFlogo* logo, bool doColor)
     sprintf(logo->chars[19], FASTFETCH_TEXT_MODIFIER_BOLT"%s                                              "FASTFETCH_TEXT_MODIFIER_RESET, color);
 }
 
+static void loadLogoFromFile(FFlogo* logo, bool doColor, char* pathToFile)
+{
+    strcpy(logo->name, "amogus");
+
+    const char* color = doColor ? "\033[91m" : "";
+    strcpy(logo->color, "\033[91m");
+
+    FILE* fp = fopen(pathToFile, "r");
+    char buff[255];
+    char* normalizedBuff;
+    int len;
+
+    int logoWidth = 0, logoHeight = 0;
+    while(!feof(fp)) {
+        fgets(buff, sizeof(buff), fp);
+        normalizedBuff = strtok(buff, "\n");
+        
+        sprintf(logo->chars[logoHeight++],   FASTFETCH_TEXT_MODIFIER_BOLT"%s%s"FASTFETCH_TEXT_MODIFIER_RESET, color, normalizedBuff);
+
+        len = utf8_strlen(normalizedBuff);
```

Comment:
Im not into this topic, why isn't a simple `strlen` enough? The point of Ascii logos is that they can be encoded using ascii so i'm not sure if we need to support unicode.

Reply:
It's not necessary, but it's a nice bonus. If you don't think there has to be support for UTF8 i can just replace it with strlen